clicache/src/CacheableBuiltins.hpp

```diff
@@ -386,20 +386,20 @@ namespace Apache
            }                                                                     \
            \
            internal:                                                               \
-           static IGeodeSerializable^ Create(native::SerializablePtr obj)            \
+           static IGeodeSerializable^ Create(std::shared_ptr<native::Serializable> obj)            \
            {                                                                     \
            return (obj != nullptr ? gcnew m(obj) : nullptr);                   \
            }                                                                     \
            \
            private:                                                                \
-             inline m(native::SerializablePtr nativeptr)                            \
+             inline m(std::shared_ptr<native::Serializable> nativeptr)                            \
               : CacheableBuiltinKey(nativeptr) { }                                \
       };
 
 
 #define _GFCLI_CACHEABLE_ARRAY_DEF_NEW(m, mt)                                    \
       ref class m : public CacheableBuiltinArray<            \
-        native::m, native::m##Ptr, mt, GeodeClassIds::m>                  \
```

Comment:
Why is this type changing away from shared_ptr?

Reply:
In the changes below this line, we went from "somethingPtr" to std::shared_ptr<"something"> var... So the ##Ptr was no longer required.

Reply:
Perhaps I'm missing something.  Why wasn't the line changed to `native::m, std::shared_ptr<native::m>, mt, GeodeClassIds::m>`?  Did you change the second parameter in the `CacheableBuiltinArray` template away from a shared_ptr?